src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -172,6 +181,11 @@ public MetricsBean(MeterRegistry meterRegistry, Environment env, TaskScheduler t
 
             registerExerciseAndExamMetrics();
             registerPublicArtemisMetrics();
+
+            if (localCIBuildJobQueueServiceOptional.isPresent()) {
+                registerLocalCIMetrics();
```

Comment:
Comment for me: We can live calculate it using registering. See Websocket stats.